ui/packages/atlasmap/src/Views/ColumnMapperView/Columns/SourcesColumn.tsx

```diff
@@ -146,23 +148,28 @@ export const SourcesColumn: FunctionComponent<
                     <Tooltip
                       position={"top"}
                       enableFlip={true}
-                      content={<div>Create a property for use in mapping</div>}
+                      content={
+                        <div>Create a source property for use in mapping</div>
+                      }
                       key={"create-property"}
                     >
                       <Button
-                        onClick={onCreateProperty}
+                        onClick={() => {
+                          onCreateSourceProperty();
```

Comment:
Should be able to just pass the `onCreateSourceProperty` reference here instead of using the lambda, no?

Reply:
yep- good catch - will fix.